backend/api/Services/MissionSchedulingService.cs

```diff
@@ -55,10 +55,17 @@ public async Task StartNextMissionRunIfSystemIsAvailable(string robotId)
                 logger.LogInformation("The robot was ready to start mission, but no mission is scheduled");
                 if (robot.MissionQueueFrozen) { return; }
 
+                if (!await localizationService.RobotIsLocalized(robotId))
+                {
+                    string infoMessage = $"Not scheduling a return to home mission as the robot {robotId} is not localized.";
```

Comment:
Can this be written directly in the LogInformation?

Reply:
I personally think it's good to reduce line length for readability.

Reply:
Don't have a strong preference on this

Reply:
Me neither, can keep it as it is then